ts/IconButton/index.tsx

```diff
@@ -4,6 +4,8 @@ import {h} from 'preact';
 import MaterialComponent from '../Base/MaterialComponent';
 import Icon from '../Icon';
 
+export {Icon as IconButtonIcon} from '../Icon';
```

Comment:
What's the purpose of this? Does it really make sense to export it from IconButton as well as Icon?

Reply:
It's not totally needed but it's nice to have and doesn't add any overhead